### DISCUSSION TITLE
How do I operate sd-webui-infinite-image-browsing outside of A1111 environment?

### DISCUSSION THREAD

**Discussion by bobpuffer1** [zanllp/infinite-image-browsing#263] [Q&A] [answered]
I think it's mentioned in the docs but I can't figure out how?

**Comment by zanllp** [comment, marked as answer]
https://github.com/zanllp/sd-webui-infinite-image-browsing/issues/47

**Comment by bobpuffer1** [comment]
Thank you for this. Sorry, I missed it in my search.
